Index: src/pages/tableGame/tableGame.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/pages/tableGame/tableGame.tsx	(date 1592524726646)
+++ src/pages/tableGame/tableGame.tsx	(date 1592524726646)
@@ -0,0 +1,15 @@
+import React from "react";
+import {Link} from "react-router-dom";
+
+export const TableGame = (props: any) =>{
+
+    const handleButton = () =>{
+        window.open("http://www.facebook.com");
+    }
+
+    return <>
+        <button onClick={handleButton}>
+            Botón
+        </button>
+    </>
+}
\ No newline at end of file
Index: src/App.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react';\nimport logo from './logo.svg';\nimport {Card} from './components/card/card.component';\nimport './App.css';\nimport {LoginPage} from \"./pages/login/login\";\nimport { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';\nimport {CreateUserPage} from \"./pages/createUser/createUser\";\nimport {LevelSelectorPage} from \"./pages/level-selector/levelSelector\";\nimport {PageNotFound} from \"./pages/pageNotFound/pageNotFound\";\n\n\nfunction App() {\n  return (\n      <>\n      <BrowserRouter >\n            <Switch>\n                <Route exact path=\"/game\" component={Card}/>\n                <Route exact path='/' component={LoginPage}/>\n                <Route exact path='/createUser' component={CreateUserPage}/>\n                <Route exact path='/lvlSelector' component={LevelSelectorPage}/>\n                <Route component={PageNotFound}/>\n            </Switch>\n        </BrowserRouter>\n      </>\n  );\n}\n\nexport default App;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/App.tsx	(revision 7c6c4c72627469b5881870483cdb8ba5d0c00621)
+++ src/App.tsx	(date 1592524726562)
@@ -7,6 +7,7 @@
 import {CreateUserPage} from "./pages/createUser/createUser";
 import {LevelSelectorPage} from "./pages/level-selector/levelSelector";
 import {PageNotFound} from "./pages/pageNotFound/pageNotFound";
+import {TableGame} from "./pages/tableGame/tableGame";
 
 
 function App() {
@@ -14,7 +15,7 @@
       <>
       <BrowserRouter >
             <Switch>
-                <Route exact path="/game" component={Card}/>
+                <Route exact path="/game" component={TableGame}/>
                 <Route exact path='/' component={LoginPage}/>
                 <Route exact path='/createUser' component={CreateUserPage}/>
                 <Route exact path='/lvlSelector' component={LevelSelectorPage}/>
